src/main/java/TaskList.java

```diff
@@ -0,0 +1,65 @@
+import java.util.ArrayList;
+
+/**
+ * The TaskList class contains the task list and
+ * has methods to modify the tasks in the list.
+ */
+public class TaskList {
+
+    private static ArrayList<Task> tasks;
```

Comment:
In the event that your Duke uses many TaskLists for a certain feature, I think that the tasks list should be implemented as a non static attribute, since one tasklist should not be used for all Duke objects created.